ansible/roles/cli_minio/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Create minio directory
+  ansible.builtin.file:
+    path: /opt/minio/
+    state: directory
+    owner: deerhide-operator
+    group: deerhide-operator
+    mode: '0755'
+  become: true
+
+- name: Found and register "arch"
+  ansible.builtin.command:
+    cmd: dpkg --print-architecture
```

Comment:
Any reason why we aren't using an Ansible fact?

Reply:
I didn't know it was possible and i don't understand how to do it right now

Reply:
We must create a role for "generic" facts to be use in other one.
like dpkg arch use in minio and docker role.

Reply:
I'll remove this and use ansible fact for the architecture